src/utils/get-keys-by-value.js

```diff
@@ -0,0 +1,5 @@
+export const getKeysByValue = (object, value) => {
```

Comment:
I find there is also another util file which says getKeyByValue, since this is a list and the previous one returns a single key, can we -
1. Try to use getKeysByValue[0] for the earlier version
2. If we need to keep both functions (I do feel its fine to keep both), can we move both to object-utils.js file ?
3. Looking at both the files, they seem related but we have 2 files for 2 functions which seems to be not needed. Moving both the functions to object-utils.js file would help in easy management in future

Reply:
Moved both the utils to object-utils.js